resource_management/CoordinationSignalHeader header
CoordinationStateColor[] states_Color
CoordinationStateOnOff[] states_OnOff
---
uint32 id